packages/backend/src/services/HealthService/HealthService.ts

```diff
@@ -67,6 +67,10 @@ export class HealthService {
                     oauth2ClientId:
                         this.lightdashConfig.auth.google.oauth2ClientId,
                 },
+                okta: {
```

Comment:
Are we going to support both google and okta simultaneously ? could this cause problems ?

Reply:
This should work well, it was built from the start for users->identities being one-to-many. Still we can test them both side-by-side in the PR preview (just setting up now)